scripts/refactor-health-check.js

```
@@ -0,0 +1,148 @@
+#!/usr/bin/env node
+
+/**
+ * This script helps migrate a service's health check implementation
+ * to use the shared utility. Run it from the root directory of a service.
+ * 
+ * Usage: node refactor-health-check.js --service <service-name>
+ * 
+ * Example: node refactor-health-check.js --service auth
+ */
+
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)